src/App/App.js

```diff
@@ -29,6 +29,7 @@ function App() {
       key={one.id}
       caption={one.caption}
       content={one.content}
+      checkbox={true}
```

Comment:
Can you please add this information to array of cards to control this value. Also cards may be state of App component for convenient workflow of react updating and rerendering.